src/webui/www/public/setlocation.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>QBT_TR(Set location)QBT_TR[CONTEXT=TransferListWidget]</title>
```

Comment:
>But then how would any new features ever get added?

AFAIK, currently string literals in HTML & js won't be available to Transifex.

using `extra_translations.h` should be the way.